### PR TITLE
GPS Interface Endpoint Tests

### PR DESCRIPTION
**Pull request by JordanChen123**
### Description
Testing the GPS API - Issue 15

### Verification

- [ ] 

### Resources

-

### CODE REVIEW

File: tests/steps/given.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    longitude: 4.5552,
+    speed: 30,
+    heading: 45,
+    };
+
+    await GPS.create(gpsData);
+});
+
+Then('the response data matches the data in the database', async function () {
+        const db = await ConnectMongoDB();
```
Comment (jahn18) [inline tests/steps/given.ts:27]: Can you please test if we still need this `ConnectMongoDB()` call given that we do it on the step where we clear the database?  🤔

**Comment by jahn18** [comment]
Great job setting up the PR and, again, well done!

File: tests/steps/given.ts (hunk 1 above)
Comment (jahn18) [reply tests/steps/given.ts]: @JordanChen123 Did the tests still pass when you removed this statement?

Comment (JordanChen123) [reply tests/steps/given.ts]: Yes the tests still passed when I removed the statement. I should've clarified it beforehand that's my bad

Comment (jahn18) [reply tests/steps/given.ts]: All good

**Review by jahn18 — APPROVED** [review]

**Pull request merged**